### PR TITLE
update legacy terms to chrysalis

### PR DESCRIPTION
**Pull request by karimkozman**

### CODE REVIEW

**Comment by karimkozman** [comment]
Screenshots in ipfs-docs-c2-migration.md are still to be updated (public node at https://ipfs.io doesn’t appear to want to retrieve the file - awaiting  DevOps to restart https://ipfs.iota.cafe/ as it might fix it to be able to take a screenshot of the last step).  

Diagrams in architecture.md need to be updated ("transaction hash" to "message ID")
data-storage-retrieve
data-storage-store

File: tangle-data-storage/architecture.md [hunk 1]
```diff
@@ -6,9 +6,9 @@
 Running an open source project, like any human endeavor, involves uncertainty and trade-offs. We hope the architecture described below helps you to deploy similar systems, but it may include mistakes, and can’t address every situation. If you have any questions about your project, we encourage you to do your own research, seek out experts, and discuss them with the IOTA community.
 :::
 
-This blueprint uses the following architecture whereby clients upload files to the API server, which sends the files to an InterPlanetary File System (IPFS) node and attaches the IPFS hashes to transactions in the Tangle.
+This blueprint uses the following architecture whereby clients upload files to the API server, which sends the files to an InterPlanetary File System (IPFS) node and attaches the IPFS hashes to messages in the Tangle.
 
-![Data Storage PoC - IOTA/IPFS - Architecture](/img/blueprints/data-storage-ipfs.png)
+![Data Storage PoC - IOTA/IPFS - Architecture](../static/img/blueprints/data-storage-ipfs.png)
```
Comment (lucas-tortora) [inline tangle-data-storage/architecture.md:11]: I know it's a bit of a pain as you can't see images outside the docusaurus build, but as the repo will be imported by the wiki when building, we need to use absolute image paths for static assets. 
```suggestion
![Data Storage PoC - IOTA/IPFS - Architecture](/img/blueprints/data-storage-ipfs.png)
```

File: tangle-data-storage/architecture.md [hunk 2]
```diff
@@ -22,51 +22,56 @@ The API server exposes two methods to the client:
...
 - Capture additional file metadata
 
 Behind the scenes, the API does the following:
 
 - Upload the file content to IPFS, which returns the IPFS hash
-- Attach the file metadata, SHA256 hash, and IPFS hash to the Tangle in a transaction, and return the transaction hash to the client
+- Attach the file metadata, SHA256/SHA3 hash, and IPFS hash to the Tangle in a message, and return the message id to the client
 
-![Data Storage PoC - IOTA/IPFS - Store File](/img/blueprints/data-storage-store.png)
+![Data Storage PoC - IOTA/IPFS - Store File](../static/img/blueprints/data-storage-store.png)
```
Comment (lucas-tortora) [inline tangle-data-storage/architecture.md:33]: I know it's a bit of a pain as you can't see images outside the docusaurus build, but as the repo will be imported by the wiki when building, we need to use absolute image paths for static assets.

```suggestion
![Data Storage PoC - IOTA/IPFS - Store File](/img/blueprints/data-storage-store.png)
```

File: tangle-data-storage/deployment-and-testing.md [hunk 3]
```diff
@@ -73,7 +73,7 @@ To deploy this application on your local network, complete the following instruc
 
 When the client connects to the API, the following page will be opened in your default web browser:
 
-![Upload File for IOTA IPFS Data Storage PoC](/img/blueprints/data-storage-upload.png)
+![Upload File for IOTA IPFS Data Storage PoC](../static/img/blueprints/data-storage-upload.png)
```
Comment (lucas-tortora) [inline tangle-data-storage/deployment-and-testing.md:76]:
```suggestion
![Upload File for IOTA IPFS Data Storage PoC](/img/blueprints/data-storage-upload.png)
```

File: tangle-data-storage/deployment-and-testing.md [hunk 4]
```diff
@@ -85,22 +85,22 @@ This page is the front end to the application, which you can use to test it.
 
 1. Select a file, and see that the other fields are automatically populated with metadata
 
-    ![Upload File for IOTA IPFS Data Storage PoC - Populated](/img/blueprints/data-storage-upload2.png)
+    ![Upload File for IOTA IPFS Data Storage PoC - Populated](../static/img/blueprints/data-storage-upload2.png)
```
Comment (lucas-tortora) [inline tangle-data-storage/deployment-and-testing.md:88]:
```suggestion
![Upload File for IOTA IPFS Data Storage PoC - Populated](/img/blueprints/data-storage-upload2.png)
```

File: tangle-data-storage/deployment-and-testing.md [hunk 5]
```diff
@@ -85,22 +85,22 @@ This page is the front end to the application, which you can use to test it.
...
 
 2. Click **Upload** to store the metadata on the IPFS node and the Tangle. If everything went well, you should see a confirmation message.
 
-    ![Uploaded File for IOTA IPFS Data Storage PoC](/img/blueprints/data-storage-upload-finished.png)
+    ![Uploaded File for IOTA IPFS Data Storage PoC](../static/img/blueprints/data-storage-upload-finished.png)
```
Comment (lucas-tortora) [inline tangle-data-storage/deployment-and-testing.md:92]:
```suggestion    
![Uploaded File for IOTA IPFS Data Storage PoC](/img/blueprints/data-storage-upload-finished.png)
```

File: tangle-data-storage/deployment-and-testing.md [hunk 6]
```diff
@@ -85,22 +85,22 @@ This page is the front end to the application, which you can use to test it.
...
 
 3. To validate a file, go to the Retrieve File page.
 
-    ![Retrieve File for IOTA IPFS Data Storage PoC](/img/blueprints/data-storage-upload-retrieve.png)
+    ![Retrieve File for IOTA IPFS Data Storage PoC](../static/img/blueprints/data-storage-upload-retrieve.png)
```
Comment (lucas-tortora) [inline tangle-data-storage/deployment-and-testing.md:96]:
```suggestion
    ![Retrieve File for IOTA IPFS Data Storage PoC](/img/blueprints/data-storage-upload-retrieve.png)
```

File: tangle-data-storage/deployment-and-testing.md [hunk 7]
```diff
@@ -85,22 +85,22 @@ This page is the front end to the application, which you can use to test it.
...
 
-4. Enter a transaction hash and click **RETRIEVE**. The transaction hash will be passed to the API, which will get the IPFS hash from the transaction's `signatureMessageFragment` field, which is used to download the file and validate its SHA256 hash against the one stored in the transaction.
+4. Enter a message ID and click **RETRIEVE**. The message ID will be passed to the API, which will get the IPFS hash from the message's `IndexationPayload` field, which is used to download the file and validate its SHA256 hash against the one stored in the message.
 
-    ![Retrieve File for IOTA IPFS Data Storage PoC](/img/blueprints/data-storage-upload-validated.png)
+    ![Retrieve File for IOTA IPFS Data Storage PoC](../static/img/blueprints/data-storage-upload-validated.png)
```
Comment (lucas-tortora) [inline tangle-data-storage/deployment-and-testing.md:100]:
```suggestion
    ![Retrieve File for IOTA IPFS Data Storage PoC](/img/blueprints/data-storage-upload-validated.png)
```

File: tangle-data-storage/deployment-and-testing.md [hunk 8]
```diff
@@ -85,22 +85,22 @@ This page is the front end to the application, which you can use to test it.
...
 
 This completes the full lifecycle for storing, retrieving, and validating a file. The file metadata, SHA256 hash and IPFS hash that are stored in a transaction in the Tangle are immutable. If the file contents that you retrieve from the IPFS node don't match the ones in the Tangle, then the contents of that file can no longer be trusted.
 
 ## Next Steps
 
-Try [running your own node](https://legacy.docs.iota.org/docs/node-software/1.0/overview) and configuring the application to connect to it.
+Try [running your own node](https://chrysalis.docs.iota.org/node_software) and configuring the application to connect to it.
```
Comment (lucas-tortora) [inline tangle-data-storage/deployment-and-testing.md:106]: We're in the process of redirecting everything to the wiki.
```suggestion
[Run your own node](https://wiki.iota.org/chrysalis-docs/node_software) and configure the application to connect to it.
```

**Review by lucas-tortora — CHANGES_REQUESTED** [review]
Great work :) !! Just some small changes.  Please let me know if you have any doubts/issues with these.

**Comment by karimkozman** [comment]
Screenshots update remaining

**Review by lucas-tortora — APPROVED** [review]
LGTME! Thanks for the updates :)

**Pull request merged**